power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/common_solver_functions.hpp

```diff
@@ -191,6 +192,7 @@ inline void calculate_pf_result(YBus<sym> const& y_bus, PowerFlowInput<sym> cons
         calculate_load_gen_result<sym>(load_gens, bus_number, input, output, load_gen_func);
         calculate_source_result<sym>(sources, bus_number, y_bus, input, output, load_gens);
     }
+    (void)std::forward<LoadGenFunc>(load_gen_func);
```

Comment:
why this?

Reply:
this function captures `func` by perfect forwarding. It is not allowed by the standard to not forward a forwarded argument, or move a moved argument.

Reply:
See https://github.com/PowerGridModel/power-grid-model/pull/882#discussion_r1938948404